ibis/backends/bigquery/tests/unit/udf/snapshots/test_usage/test_udf_sql/out.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TEMPORARY FUNCTION format_t_0(input STRING)
+RETURNS FLOAT64
+AS ([FORMAT('%T', input));
```

Comment:
Is this correct? It looks like there's an unmatched `[` character before `FORMAT`.